core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -114,4 +114,24 @@ package object config {
   private[spark] val PYSPARK_PYTHON = ConfigBuilder("spark.pyspark.python")
     .stringConf
     .createOptional
+
+  private[spark] val SPARK_IO_ENCRYPTION_ENABLED = ConfigBuilder("spark.io.encryption.enabled")
+    .booleanConf
+    .createWithDefault(false)
+
+  private[spark] val SPARK_IO_ENCRYPTION_KEYGEN_ALGORITHM = ConfigBuilder(
+    "spark.io.encryption.keygen.algorithm")
```

Comment:
nit: follow this pattern

https://github.com/apache/spark/pull/8880/files#diff-6bdad48cfc34314e89599655442ff210R101